returnn/tf/util/data.py

```diff
@@ -2956,8 +2956,8 @@ def copy_template_unpack_batch(self):
             dim_tags.append(virtual_dim.dim_tag)
             batch = batch.copy_remove_dim(virtual_dim)
           elif not new_batch_dim_tag:
-            new_batch_dim_tag = Dim(kind=Dim.Types.Batch)
-            dim_tags.append(new_batch_dim_tag)
+            new_batch_dim_tag = dim_tag
+            dim_tags.append(dim_tag)
```

Comment:
But it should be `batch_dim`, right?
```suggestion
            new_batch_dim_tag = batch_dim
            dim_tags.append(new_batch_dim_tag)
```

Reply:
`batch_dim` is not an existing variable, not sure what you mean here?

Reply:
You need to rebase.

Reply:
I already did, now again with the current master. Still not seeing `batch_dim`.

Reply:
[Here](https://github.com/rwth-i6/returnn/blob/4195c3fe3b2bf7871cdb33779df5fbf0792bbd0b/returnn/tf/util/data.py#L813) is `batch_dim`. Why don't you see that?

Reply:
I expected it to be a local variable. Sorry. Added your suggestion now.